<launch>
  <node name="segment_plane" pkg="calibration_point_clouds" type="segment_plane_node">
    <param name="loop_rate" value="10" />
    <param name="inliers_pc_topic_name" value="/inliers_pointcloud" />
    <param name="source_pc_topic_name" value="/kinect_first/kinect2/hd/points" />
    <param name="inliers_pc_frame_id" value="/kinect_first/kinect2_rgb_optical_frame" />
    <param name="leaf_size_x" value="0.1" />
    <param name="leaf_size_y" value="0.1" />
    <param name="leaf_size_z" value="0.1" />
  </node>
</launch>
